docs/source/awesome-trio-libraries.rst

```diff
@@ -0,0 +1,64 @@
+* `trio-websocket <https://github.com/HyperionGray/trio-websocket>`__ - This library implements the WebSocket protocol, striving for safety, correctness, and ergonomics.
+* `quart-trio <https://gitlab.com/pgjones/quart-trio/>`__ - Like Flask, but for Trio. A simple and powerful framework for building async web applications and REST APIs. Tip: this is an ASGI-based framework, so you'll also need an HTTP server with ASGI support.
+* `hypercorn <https://gitlab.com/pgjones/hypercorn>`__ - An HTTP server for hosting your ASGI apps. Supports HTTP/1.1, HTTP/2, HTTP/3, and Websockets. Can be run as a standalone server, or embedded in a larger Trio app. Use it with ``quart-trio``, or any other Trio-compatible ASGI framework.
+
+
+Database
+--------
+
+* `triopg <https://github.com/python-trio/triopg>`__ - PostgreSQL client for Trio based on asyncpg
+* `trio-mysql <https://github.com/python-trio/trio-mysql>`__ - Pure Python MySQL Client
+* `sqlalchemy_aio <https://github.com/RazerM/sqlalchemy_aio>`__ - Add asyncio and Trio support to SQLAlchemy core, derived from alchimia.
+* `sphinxcontrib-trio <https://github.com/python-trio/sphinxcontrib-trio>`__ - Make Sphinx better at documenting Python functions and methods
+
+Building Command Line Apps
+--------------------------
+* `trio-click <https://github.com/python-trio/trio-click>`__ - Python composable command line utility, trio-compatible version
+
+Multi-Core/Multiprocessing
+--------------------------
+* `tractor <https://github.com/goodboy/tractor>`__ - tractor is an attempt to bring trionic structured concurrency to distributed multi-core Python
+
+
+Testing
+-------
+* `pytest-trio <https://github.com/python-trio/pytest-trio>`__ - Pytest plugin for trio
+* `hypothesis-trio <https://github.com/python-trio/hypothesis-trio>`__ - Hypothesis plugin for trio
+* `trustme <https://github.com/python-trio/trustme>`__ - #1 quality TLS certs while you wait, for the discerning tester
+
+
+Tools and Utilities
+-------------------
+* `trio-typing <https://github.com/python-trio/trio-typing>`__ - Type hints for Trio and related projects
+* `trio-util <https://github.com/groove-x/trio-util>`__ - An assortment of utilities for the Trio async/await framework
+* `tricycle <https://github.com/oremanj/tricycle>`__ - This is a library of interesting-but-maybe-not-yet-fully-proven extensions to Trio
+
+Trio/Asyncio Interoperability
+-----------------------------
+* `anyio <https://github.com/agronholm/anyio>`__ - AnyIO is a asynchronous compatibility API that allows applications and libraries written against it to run unmodified on asyncio, curio and trio
+* `sniffio <https://github.com/python-trio/sniffio>`__ - This is a tiny package whose only purpose is to let you detect which async library your code is running under
+* `trio-asyncio <https://github.com/python-trio/trio-asyncio>`__ - Trio-Asyncio is a re-implementation of the asyncio mainloop on top of Trio
```

Comment:
While that's true, I think people will get a better idea of why they should care about trio-asyncio if we describe it as "Trio-Asyncio lets you use asyncio libraries from your Trio app."

Reply:
**many** asyncio libraries. Not all of them, aiohttp being the most glaring counterexample.